testing/constraints-3.6.txt

```diff
@@ -0,0 +1,16 @@
+fastparquet==0.4.1
```

Comment:
The minimum is not specified, thus I used the current latest version, i.e. the one that gets installed right now by default.